public/js/create-page.js

```diff
@@ -1,13 +1,16 @@
 const { h, compile } = Vue;
 
-const createPage = (name, data = {}, methods = {}, updated = () => {}) => Vue.defineComponent({
+const createPage = (name, data = {}, inject = [], computed = {}, provide = null, methods = {}, hooks = {}) => Vue.defineComponent({
```

Comment:
I didn't check, but new params should be after last one as can possibly break existing arg call used as "updated" param fill:

https://github.com/samsonasik/ci4-vue/blob/24aa7f78e95a0b2f0022e27264aaa44dcf9128a2/public/js/portfolio.js#L51

Reply:
My bad I didn't update the protfolio.js. Will do in a moment!